src/diffusers/pipelines/animatediff/pipeline_animatediff_video2video.py

```diff
@@ -917,6 +886,14 @@ def __call__(
         latent_timestep = timesteps[:1].repeat(batch_size * num_videos_per_prompt)
 
         # 5. Prepare latent variables
+        # video must be a list of list of images
+        # the outer list denotes having multiple videos as input, whereas inner list means the frames of the video
+        # as a list of images
+        if video and not isinstance(video[0], list):
+            video = [video]
+        if latents is None:
+            video = self.video_processor.preprocess_video(video, height=height, width=width, preceed_with_frames=True)
```

Comment:
we do not need this because this pipeline currently only accept video as images (not tensor or array) 
```suggestion
            video = self.video_processor.preprocess_video(video, height=height, width=width)
```

Reply:
we do not need this flag because :
1. I think the requirement is the same for all the video pipelines because the vae always expects same shape for the input; i.e. animatediff is not different from text-to-video (text-to-video permutate `video` input again before it puts the video through the encoder https://github.com/huggingface/diffusers/blob/b934215d4c376ea2e08e28103443686b95ea772c/src/diffusers/pipelines/text_to_video_synthesis/pipeline_text_to_video_synth_img2img.py#L522);
2. even if there is a difference between pipelines, we should handle that difference inside the pipeline and we should always return the same shape from the video processor, whether it is channel before frame or frame before channel